website/content/partials/components/platform-nomad-jobspec-canary.mdx

```diff
@@ -113,6 +115,8 @@ The Vault token used to deploy the Nomad job with a token having specific Vault
 
 Uses the runner config environment variable VAULT_TOKEN.
```

Comment:
Prior to the environment variables being presented in the website, part of feedback on one of my PRs was to specify that the configuration was read from an environment variable in the description of the config. 

In a new PR, I think that part of the description should be removed, since the environment variable will be specified as a new field now! 😄